### ISSUE TITLE
New term - papillary cystadenoma

### ISSUE THREAD

**Issue by kkrysiak** [obophenotype/human-phenotype-ontology#3743]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
papillary cystadenoma

### 2. Synonyms
papillary cystadenofibroma

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
Benign tumor of the epididymis in men and the broad ligament in women. PMID: 11493991
More info here as well: http://www.pathologyoutlines.com/topic/testisepidpapcyst.html

### 4. Parent term (use [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
I'm not sure because HP:0009715 is an example of one. But the female counterpart specific to the broad ligament in women is not in HPO.

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
von Hippel Lindau (OMIM:193300)

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))
0000-0002-6299-9230

**Comment by pnrobinson** [comment]
This is the term:
https://hpo.jax.org/app/browse/term/HP:0009715
